server.js

```diff
@@ -21,35 +21,37 @@ app.use(express.static('public'));
 // body parser config to accept our datatypes
 app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
If you are going to use postman to check any of your routes, you'll want to make sure to configure bodyParser to work with JSON. Look back to yesterdays code for an example.